### ISSUE TITLE
How to train the optical flow estimate module (RAFT) ?

### ISSUE THREAD

**Issue by dodododddo** [TomTomTommi/bidavideo#5]
Great work! I'd like to train a BiDAStablizer on DNA-Rendering Dataset. However,  DNA-Rendering has no optical flow gt. I found that some of the datasets motioned in the paper do not have optical flow gt as well, so how to train BiDAStablizer in such a situation? Just use pretrain RAFT and use end2end loss?

**Comment by TomTomTommi** [comment]
Exactly. We do not use gt optical flow in the training process.

**Comment by dodododddo** [comment]
Is it convenient to provide the checkpoint of the pre-trained optical flow model you use?

**Comment by TomTomTommi** [comment]
just the official checkpoints, e.g., [RAFT](https://github.com/princeton-vl/RAFT)
